NextcloudUITests/NextcloudUITests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  NextcloudUITests.swift
+//  NextcloudUITests
+//
+//  Created by Milen Pivchev on 5/19/23.
+//  Copyright © 2023 Marino Faggiana. All rights reserved.
+//
+
+import XCTest
+
+final class NextcloudUITests: XCTestCase {
+    override class var runsForEachTargetApplicationUIConfiguration: Bool {
+        false
+    }
```

Comment:
If this is `true` the UI tests are not ran, it seems as if there are no targets. Not sure why this is happening, but probably a problem with our project configuration or an XCode bug.